From fd4676f543c310778aae3ca78c2aaa6e186e1056 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:09:22 -0400
Subject: [usb] host/xhci-plat: sort the headers in alphabetic order

Message-id: <1410887463-20674-491-git-send-email-dzickus@redhat.com>
Patchwork-id: 95642
O-Subject: [RHEL7 PATCH 490/591] usb: host: xhci-plat: sort the headers in alphabetic order
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 48157bb97f074d21372bd3ae87e5988ed23c8972
Author: Gregory CLEMENT <gregory.clement@free-electrons.com>
Date:   Thu May 15 12:17:31 2014 +0200

    usb: host: xhci-plat: sort the headers in alphabetic order

    Sorting the headers in alphabetic order will help to reduce the conflict
    when adding new headers later.

    Signed-off-by: Gregory CLEMENT <gregory.clement@free-electrons.com>
    Acked-by: Felipe Balbi <balbi@ti.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/host/xhci-plat.c b/drivers/usb/host/xhci-plat.c
index 30ae396..5c11e3e 100644
--- a/drivers/usb/host/xhci-plat.c
+++ b/drivers/usb/host/xhci-plat.c
@@ -11,11 +11,11 @@
  * version 2 as published by the Free Software Foundation.
  */
 
-#include <linux/platform_device.h>
+#include <linux/dma-mapping.h>
 #include <linux/module.h>
-#include <linux/slab.h>
 #include <linux/of.h>
-#include <linux/dma-mapping.h>
+#include <linux/platform_device.h>
+#include <linux/slab.h>
 
 #include "xhci.h"
 
-- 
1.7.1